private/pkg/storage/diff.go

```diff
@@ -278,6 +296,7 @@ type diffOptions struct {
 	externalPaths         bool
 	oneExternalPathPrefix string
 	twoExternalPathPrefix string
+	transforms            []func(side, filename string, content []byte) []byte
```

Comment:
nit for consistency:
```suggestion
	transforms            []func(side string, filename string, content []byte) []byte
```